styles.css

```diff
@@ -0,0 +1,154 @@
+body {
+  margin: 0 auto;
+  background-color: #1c1a19;
+  background-size: cover;
+}
+
+.mainMenu {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  background-color: #3c3a39;
+}
+
+.menuLogo {
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  padding-left: 4%;
+}
+
+.menuIcon {
+  position: relative;
+  top: 12px;
+  width: 22px;
+  right: 24px;
+}
+
+.menuLvl {
+  display: block;
+  background-color: #fff;
+  width: 100%;
+  height: 2.5px;
+  margin: 4px;
+  border-radius: 2px;
+}
+
+.homeContent {
+  padding-left: 16%;
+  padding-right: 16%;
+  background-image: url(./images/illustration-header-mobile.png);
+  background-repeat: no-repeat;
+  background-position-y: 100%;
+  background-position-x: 100%;
+}
```

Comment:
- [ ] Dear @RuriEnciso, It's among the primary requirement to use flex-box to align contents inside this section. This gives us a lot of options to align all our elements, plus, it's responsive.